src/lib/dlt_user.c

```diff
@@ -1838,7 +1838,7 @@ DltReturnValue dlt_user_log_write_start_internal(DltContext *handle,
         return DLT_RETURN_WRONG_PARAMETER;
     } else if (ret == DLT_RETURN_LOGGING_DISABLED) {
         log->handle = NULL;
-        return DLT_RETURN_OK;
```

Comment:
Would it be meaning that the behavior of some scenarios when logging disabled should be by-passed here?
I believe there could be some reasons behind doing this.

Reply:
> Would it be meaning that the behavior of some scenarios when logging disabled should be by-passed here?

I didn't find such a thing but maybe I just missed it. But I'm not sure if it's a good idea to abuse it like this.
If we keep return OK here it should be clearly documented that log->buffer is only initialized when log->handle is not null. We had a crash because an application assumed that the buffer is set up when ok is returned and did not check log->handle.